server/routes/users.ts

```diff
@@ -109,4 +110,19 @@ router.delete('/', async (req, res) => {
     }
 });
 
+//GET /:id
+router.get('/:userId', async (req, res) => {
```

Comment:
remove

Reply:
Same thing as above this should be gone now